pubspec.yaml

```diff
@@ -1,13 +1,12 @@
 name: http
-version: 0.12.0+4
+version: 0.12.1-dev
 homepage: https://github.com/dart-lang/http
 description: A composable, multi-platform, Future-based API for HTTP requests.
 
 environment:
   sdk: ">=2.4.0 <3.0.0"
 
 dependencies:
-  async: ">=1.10.0 <3.0.0"
```

Comment:
One less dependency! 💯

Reply:
In practice pretty much everyone is going to pick up `async` transitively, but yeah it's always nice to make the graph a little less connected 😄